description.md

```diff
@@ -17,6 +17,121 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Quizzes
 
+#### Definition
+
+Quizzes are questions that verify specific information and must be verifiable automatically.
+Therefore, multiple-choice questions are regarded as quizzes, while open-ended ones are not.
+Quizzes may be part of practical sessions, lectures or standalone as forms of evaluation.
+
+#### Purpose
+
+Quizzes provide a means to continuously evaluate students.
+This creates a positive feedback loop between trainers and trainees whereby both can track the students' progress in real time.
+
+Furthermore, quizzes serve as practice items for students to solve at their own pace.
+They may also be used for evaluation items that contribute to the students' overall grade.
+
+Another desired aspect is that quizzes be easily extractable and convertible between formats, as described [below](#deployment), and repurposed by other trainers.
+
+Despite all of their strengths, quizzes also come with a few shortcomings.
+One such downside is the binary nature of the feedback loop.
+Because multiple-choice answers may be either correct or wrong, there is little room for nuance regarding the feedback that students receive upon answering incorrectly.
+Similarly, quizzes cannot be easily tailored and personalized for the individual needs of each student.
+In addition, non-technical fields of study may find little benefit in quizzes as they rely more on arguments essays rather than concrete, technical information.
+
+#### Deployment
```

Comment:
Before this, add a summary of steps related to quizzes: brainstorm the question, write it down using the OEH format, store in the proper location in the repository, convert it to the required publisher format, publish / deploy it.

I would then detail each step in a subsection.

Reply:
Isn't this too much detail? The steps themselves are self-explanatory. For example, the whole section on Quizzes explains how to store them in the OEH format, so I don't think we should go into that here.